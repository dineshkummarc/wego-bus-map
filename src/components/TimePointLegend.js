import React from 'react';

function TimePointLegend() {
  return(
    <div className="small">
      <ul className="list-inline">
        <li className="list-inline-item mb-2"><span className="px-1 border">0:00 AM</span> Time from static schedule information.</li>
        <li className="list-inline-item mb-2"><strike className="text-muted px-1 border">0:00 AM</strike> Static scheduled time has been updated.</li>
        <li className="list-inline-item mb-2"><strong className="text-primary px-1 border">0:00 AM</strong> Updated with realtime trip information.</li>
      </ul>
    </div>
  );
}

export default TimePointLegend;
