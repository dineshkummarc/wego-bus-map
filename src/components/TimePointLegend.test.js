/* globals test, expect */

import React from 'react';
import TimePointLegend from './TimePointLegend';
import { render } from '@testing-library/react';

test('renders TimePointLegend', () => {
  const {container} = render(
    <TimePointLegend />
  );
  expect(container).toMatchSnapshot();
});
