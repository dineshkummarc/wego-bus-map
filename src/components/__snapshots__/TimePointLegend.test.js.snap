// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TimePointLegend 1`] = `
<div>
  <div
    class="small"
  >
    <ul
      class="list-inline"
    >
      <li
        class="list-inline-item mb-2"
      >
        <span
          class="px-1 border"
        >
          0:00 AM
        </span>
         Scheduled time.
      </li>
      <li
        class="list-inline-item mb-2"
      >
        <strike
          class="text-muted px-1 border"
        >
          0:00 AM
        </strike>
         Previous scheduled time.
      </li>
      <li
        class="list-inline-item mb-2"
      >
        <strong
          class="text-primary px-1 border"
        >
          0:00 AM
        </strong>
         Updated time from tracking.
      </li>
      <li
        class="list-inline-item mb-2"
      >
        <strong
          class="px-1 border"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-calendar-day fa-fw px-3"
            data-icon="calendar-day"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M160 32V64H288V32C288 14.33 302.3 0 320 0C337.7 0 352 14.33 352 32V64H400C426.5 64 448 85.49 448 112V160H0V112C0 85.49 21.49 64 48 64H96V32C96 14.33 110.3 0 128 0C145.7 0 160 14.33 160 32zM0 192H448V464C448 490.5 426.5 512 400 512H48C21.49 512 0 490.5 0 464V192zM80 256C71.16 256 64 263.2 64 272V368C64 376.8 71.16 384 80 384H176C184.8 384 192 376.8 192 368V272C192 263.2 184.8 256 176 256H80z"
              fill="currentColor"
            />
          </svg>
        </strong>
         Time spans into the next day.
      </li>
    </ul>
  </div>
</div>
`;
